app/controllers/partners/children_controller.rb

```diff
@@ -35,6 +35,11 @@ def show
     def new
       family = current_partner.families.find_by!(id: params[:family_id])
       @child = family.children.new
+
+      requestable_items = PartnerFetchRequestableItemsService.new(partner_id: current_partner.partner.id).call
+      @formatted_requestable_items = requestable_items.map do |rt|
+        [rt.name, rt.id]
+      end
```

Comment:
should we add this as a `format` method somewhere? then we can reuse between new and edit. We could combine it with the call to `PartnerFetchRequestableItemsService` and have a `fetch_and_format_requestable_items` in this controller maybe?